### PR TITLE
chore: deprecate this repo

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

**Comment by UdjinM6** [comment]
cc @thephez

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # gitian.sigs
+
+DEPRECATED: We no longer use Gitian to produce deterministic binaries, we use Guix instead.
+You can find corresponding signatures [here](https://github.com/dashpay/guix.sigs).
```
Comment (thephez) [inline README.md:4]: Nit
```suggestion
DEPRECATED: We no longer use Gitian to produce deterministic binaries; we use [Guix](https://guix.gnu.org/) instead.
You can find corresponding signatures in the [guix.sigs repository](https://github.com/dashpay/guix.sigs).
```

Comment (UdjinM6) [reply README.md]: Applied, thanks!

**Review by thephez — APPROVED** [review]
LGTM

**Comment by schinzelh** [comment]
The end of a era - I remember doing the first gitian builds in 2014 to prevent Evan from distributing his "Notebook builds". Precious memories :)

LGTM

**Review by knst — APPROVED** [review]
LGTM.

**Review by PastaPastaPasta — APPROVED** [review]

**Pull request merged**